src/routes/Generator.svelte

```diff
@@ -38,24 +38,43 @@
 			};
 	}
 
-	async function exportArt() {
+	async function renderArt() {
 		const { Image } = await import('image-js');
 		const canvas = await html2canvas(albumArt);
 
-		let image = await Image.load(canvas.toDataURL('image/jpeg'));
+		let image = await Image.load(canvas.toDataURL('image/png'));
 		const originalWidth = image.width;
 		image = image
 			.resize({ preserveAspectRatio: true, width: originalWidth * 0.85 }) // Get those compression artifacts like the original cover
 			.resize({ preserveAspectRatio: true, width: originalWidth })
-			.blurFilter({ radius: 2 });
+			.blurFilter({ radius: 3 });
```

Comment:
I'm not really getting along with this blur radius. 

This is with radius = 2
![image](https://github.com/user-attachments/assets/f2505bd6-e614-4e1f-99ca-a51cccc38e93)

This is with radius = 3
![image](https://github.com/user-attachments/assets/c68abfb9-04ff-485c-ad54-d15ea1492c41)

The increase in blur is pretty significant for smaller text. For reference, I had checked the deluxe album cover art for the blur radius. I think we could keep the radius as-is.

Reply:
How do we settle this conflict ? Maybe set to 2.5 default and make is a variable for the ADVANCED USERS ??

Reply:
> How do we settle this conflict ? Maybe set to 2.5 default and make is a variable for the ADVANCED USERS ??

@Ellipse0934 we could indeed have an option to set both the blur radius and the "compressed image" effect but I'm not looking to complicate that and instead keep the result as close to the original album art as possible.